pontoon/teams/templates/teams/includes/info.html

```diff
@@ -1,5 +1,36 @@
-{% if locale.team_description %}
-  <div class="info">{{ locale.team_description|safe }}</div>
-{% else %}
-  <p class="no-results">No team info available.</p>
-{% endif %}
+<div id="info-wrapper">
+  {% if request.user.has_perm('base.can_manage_locale', locale) %}
+  <div class="controls">
+    <a class="edit-info button" href="#"><span class="fa fa-pencil"></span>Edit</a>
+  </div>
+  {% endif %}
+
+  <div class="read-only-info">
+    <div class="info">
+    {% if locale.team_description %}
+      {{ locale.team_description|safe }}
+    {% else %}
+      <p class="no-results">No team info available.</p>
+    {% endif %}
+    </div>
+  </div>
```

Comment:
There are 2 issues with locales without `team_description`:
* If you edit info for such locale, you get `<p class="no-results">No team info available.</p>` in the textarea.
* If you save empty team info, the `No team info available.` text doesn't show up.

I suggest that you keep using the same HTML as we do for project info, but keep both HTML elements in the DOM:
```jinja
<div class="info">{{ locale.team_description|safe }}</div>
<p class="no-results {% if locale.team_description %}hidden{% endif %}">No team info available.</p>
```
Then in the `success` function of the `$.ajax call in JS toggle both elements based on the returned data:
```js
container.find('.info').html(data).toggle(data !== '');
container.find('.no-results').toggle(data === '');
```